src/hooks/useGetMoviesById.tsx

```diff
@@ -0,0 +1,39 @@
+import { useQueries } from "@tanstack/react-query";
+import axios from "axios";
+import { Movie } from "../types/movie";
+
+export const useGetMoviesById = () => {
```

Comment:
Please make this function take an array of IDs as an argument. Don't hard-code the ID-array.